src/current/v24.2/create-changefeed.md

```diff
@@ -184,7 +184,7 @@ Option | Value | Description
 <a name="cursor"></a>`cursor` | [Timestamp]({% link {{ page.version.version }}/as-of-system-time.md %}#parameters)  | Emit any changes after the given timestamp. `cursor` does not output the current state of the table first. When `cursor` is not specified, the changefeed starts by doing an initial scan of all the watched rows and emits the current value, then moves to emitting any changes that happen after the scan.<br><br>The changefeed will encounter an error if you specify a timestamp that is before the configured garbage collection window for the target table. (Refer to [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#replication-zone-variables).) With default garbage collection settings, this means you cannot create a changefeed that starts more than [the-default-MVCC-garbage-collection-interval]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) in the past.<br><br>You can use `cursor` to [start-a-new-changefeed-where-a-previous-changefeed-ended](#start-a-new-changefeed-where-another-ended).<br><br>Example: `cursor='1536242855577149065.0000000000'`
 <a name="diff"></a>`diff` | N/A |  Publish a [`before` field]({% link {{ page.version.version }}/changefeed-messages.md %}#wrapped-and-diff) with each message, which includes the value of the row before the update was applied. Changefeeds must use the `diff` option with the default [`wrapped` envelope](#envelope) to emit the `before` field.
-<a name="encode-json-value-null-as-object"></a><span class="version-tag">New in v24.2:</span> `encode_json_value_null_as_object` | N/A | Emit JSON `NULL` values as `{"__crdb_json_null__": true}` to distinguish these values from SQL `NULL` values. Refer to the [Changefeed Messages]({% link {{ page.version.version }}/changefeed-messages.md %}#json) page for an example.<br><br>**Note:** When this option is enabled, if the changefeed encounters the literal value `{"__crdb_json_null__": true}` in JSON, it will have the same representation as a JSON `NULL` value and a warning will be printed to the [`DEV` logging channel]({% link {{ page.version.version }}/logging.md %}#dev).
+<a name="encode-json-value-null-as-object"></a><span class="version-tag">New in v24.2:</span> `encode_json_value_null_as_object` | N/A | Emit JSON `NULL` values as `{"__crdb_json_null__": true}` to distinguish these values from SQL `NULL` values. Refer to the [Changefeed Messages]({% link {{ page.version.version }}/changefeed-messages.md %}) page for an example.<br><br>**Note:** When this option is enabled, if the changefeed encounters the literal value `{"__crdb_json_null__": true}` in JSON, it will have the same representation as a JSON `NULL` value and a warning will be printed to the [`DEV` logging channel]({% link {{ page.version.version }}/logging.md %}#dev).
```

Comment:
```suggestion
<a name="encode-json-value-null-as-object"></a><span class="version-tag">New in v24.2:</span> `encode_json_value_null_as_object` | N/A | Emit JSON `NULL` values as `{"__crdb_json_null__": true}` to distinguish these values from SQL `NULL` values. Refer to the [Changefeed Messages]({% link {{ page.version.version }}/changefeed-messages.md %}#json) page for an example.<br><br>**Note:** When this option is enabled, if the changefeed encounters the literal value `{"__crdb_json_null__": true}` in JSON, it will have the same representation as a JSON `NULL` value and a warning will be printed to the [`DEV` logging channel]({% link {{ page.version.version }}/logging.md %}#dev).
```
The link will work when we merge `main` to get https://github.com/cockroachdb/docs/pull/18718.